src/privPrecCompare.ml

```diff
@@ -1,45 +1,6 @@
-open! Defaults (* CircInterval / Enums / ... need initialized conf *)
-open! Batteries
-open Prelude
-open Ana
-open PrivPrecCompareUtil
-open PrecCompare
-
-module VD = BaseDomain.VD
-
-let load filename =
-  let f = open_in_bin filename in
-  let dump: dump = Marshal.from_channel f in
-  close_in_noerr f;
-  dump
-
-module CompareDump = MakeHashtbl (LV) (VD) (LVH)
-
-let compare_dumps {name = name1; lvh = lvh1} {name = name2; lvh = lvh2} =
-  CompareDump.compare ~name1 lvh1 ~name2 lvh2
-
-let count_locations dumps =
-  let module LH = Hashtbl.Make (CilType.Location) in
-  let locations = LH.create 113 in
-  let location_vars = LVH.create 113 in
-  List.iter (fun {lvh; _} ->
-      LVH.iter (fun (l, x) _ ->
-          LH.replace locations l ();
-          LVH.replace location_vars (l, x) ()
-        ) lvh
-    ) dumps;
-  (LH.length locations, LVH.length location_vars)
+(* Utility program to compare precision of results of the base analysis with different privatizations *)
+(* The result files that can be compared here may be created by passing an output file name in "exp.priv-prec-dump" to Goblint *)
+module A = PrePrivPrecCompare.ComparePrec (PrivPrecCompareUtil.Util)
```

Comment:
`Prec` and `Compare` are named twice in `PrePrivPrecCompare.ComparePrec`. Maybe just move and rename it to `PrecCompare.Make` to avoid the unnecessary duplication and also remove `Priv` since the Apron version isn't priv-specific anymore.

Reply:
@sim642 There is already a module `PrecCompare.Make`. Should one just move that into  `PrecCompare.Comparison.Make`?

Reply:
I think not, it doesn't completely make sense there because the `Comparison` module is just for that type.

Maybe `MakeDump` or something then, because the content of `ComparePrec` has to do with dumping and comparing the dumps.

Reply:
Ok, changed it.